pkg/api/validation/validation.go

```diff
@@ -2785,61 +2786,68 @@ func validateServicePort(sp *api.ServicePort, requireName, isHeadlessService boo
 	return allErrs
 }
 
-// validateServiceExternalTrafficFields validates ExternalTraffic related annotations
+// validateServiceExternalTrafficFieldsValue validates ExternalTraffic related annotations
 // have legal value.
-func validateServiceExternalTrafficFields(service *api.Service) field.ErrorList {
+func validateServiceExternalTrafficFieldsValue(service *api.Service) field.ErrorList {
```

Comment:
Not sure if it is relevant. 
Is it allowed to use healthcheck node port with Global?
Is it allowed to mixa usage of beta annotation and fields? (I think annotaion is preempting, so we do not care? )